icu4c/source/i18n/reldatefmt.cpp

```diff
@@ -809,11 +810,16 @@ RelativeDateTimeFormatter::RelativeDateTimeFormatter(
         return;
     }
     if (capitalizationContext == UDISPCTX_CAPITALIZATION_FOR_BEGINNING_OF_SENTENCE) {
+#if !UCONFIG_NO_BREAK_ITERATION
         BreakIterator *bi = BreakIterator::createSentenceInstance(locale, status);
         if (U_FAILURE(status)) {
             return;
         }
         init(nfToAdopt, bi, status);
+#else
+        status = U_UNSUPPORTED_ERROR;
+        return;
```

Comment:
nit: unnecessary line

Reply:
Markus, I think it is unnecessary if no one ever will add more lines after the else block. But would that be safer for him to put the return here in case years later someone may add some more line after the else block?

Reply:
I called it a "nit" indicating that I could live with it as it is, but it jumped out as weird and unnecessary.

It *could* be safer to return immediately when we think we are done, or it *could* be a subtle bug on a future change that intends for something else to happen at the end of the function even when there is an error. We won't know until someone wants to add stuff there.